diff --git a/build/linux/system.gyp b/build/linux/system.gyp~
index 49acd65..8a4d66c 100644
--- a/build/linux/system.gyp
+++ b/build/linux/system.gyp
@@ -175,9 +175,6 @@
                          'gps_open',
                          'gps_close',
                          'gps_read',
-                         # We don't use gps_shm_read() directly, just to make
-                         # sure that libgps has the shared memory support.
-                         'gps_shm_read',
               ],
               'message': 'Generating libgps library loader.',
               'process_outputs_as_sources': 1,
