diff --git a/third_party/WebKit/Source/WebCore/bindings/scripts/IDLAttributes.txt b/third_party/WebKit/Source/WebCore/bindings/scripts/IDLAttributes.txt~
index 1861118..3455db2 100644
--- a/third_party/WebKit/Source/WebCore/bindings/scripts/IDLAttributes.txt
+++ b/third_party/WebKit/Source/WebCore/bindings/scripts/IDLAttributes.txt~
@@ -114,6 +114,7 @@ TypedArray=*
 URL
 V8CacheAttributeForGC
 V8Custom
+V8CustomIndexedGetter
 V8CustomConstructor
 V8CustomGetter
 V8CustomIsReachable
